Specs/Scene/ModelExperimental/Model3DTileContentSpec.js

```diff
@@ -545,96 +1363,245 @@ describe(
             expect(content.hasProperty(featureId, "id")).toBe(true);
             expect(content.getFeature(featureId)).toBeDefined();
           });
-        }
-      );
+        });
+      });
+
+      // only have one destroy test (this one)
+      it("destroys glTF content", function () {
```

Comment:
FYI there's a "destroy pnts content" test, so either this comment or that test should be removed to be consistent

Reply:
And if we're keeping this comment, maybe reword this more formally...
```suggestion
      // Since all content types are loaded as a Model3DTileContent, only one destroy test is necessary
      it("destroys", function () {
```
... and move the test outside the `glTF` suite

Reply:
ah I forgot to put `TODO` in the comment so I missed a spot. 

This comment was saying keep the glTF destroy test, but remove the pnts/b3dm/i3dm equivalents. I updated these.